py/circuitpy_mpconfig.mk

```diff
@@ -163,6 +166,12 @@ CFLAGS += -DCIRCUITPY_RTC=$(CIRCUITPY_RTC)
 CIRCUITPY_SAMD ?= 0
 CFLAGS += -DCIRCUITPY_SAMD=$(CIRCUITPY_SAMD)
 
+CIRCUITPY_SDCARDIO ?= $(CIRCUITPY_FULL_BUILD)
+CFLAGS += -DCIRCUITPY_SDCARDIO=$(CIRCUITPY_SDCARDIO)
+
+CIRCUITPY_SDCARDIO_SDIO ?= 0
+CFLAGS += -DCIRCUITPY_SDCARDIO_SDIO=$(CIRCUITPY_SDCARDIO_SDIO)
+
```

Comment:
@tannewt is this way of making _sdcardio.SDIOSDCard "optional" going to fly, or does it need to be 2 modules?  When this define is 0, a stub constructor which throws an error is included instead.

Reply:
I'd prefer two modules. That way it'll fit into the support matrix and its related machinery. Maybe `sdioio` :-P ?